tests/phpunit/elementor/core/kits/test-manager.php

```diff
@@ -21,6 +21,40 @@ public function test_create_default_kit() {
 		$this->assertEquals( $manager->get_active_id(), $id );
 	}
 
+	public function test_create_new_kit() {
+		// Arrange
+		$manager = Plugin::$instance->kits_manager;
+
+		// Act
+		$id = $manager->create_new_kit();
+
+		// Assert
+		$this->assertEquals( $manager->get_active_id(), $id );
+	}
+
+	public function test_create_new_kit_with_title_and_settings() {
+		// Arrange
+		$manager = Plugin::$instance->kits_manager;
+
+		// Act
+		$id = $manager->create_new_kit( 'test_title', ['test_setting' => 'test_setting'] );
+
+		// Assert
+		$this->assertEquals( $manager->get_active_id(), $id );
```

Comment:
Add 2 new checks please:
1. Assert the title + settings
2. Assert that when there is no title, the title is 'Custom' (our default)